src/components/profile-header.tsx

```diff
@@ -6,18 +6,29 @@ import Button from "./button"
 const ProfileHeader = (props) => {
 
   return (
-    <div style={{display: "flex", flexDirection: "row"}}>
-        <ProfilePicture name={`${props.name}`} slug={`${props.title.toLowerCase()}`} />
-        <div sx={{ display: "flex", 
-                alignItems: "center", 
-                paddingLeft: "25px", 
-                paddingRight: "25px" 
-        }}>
-            <h2 sx={{ fontSize: [4, 5, 6], fontWeight: `bold`, color: `title` }}>
-                {props.title}
-            </h2>
-        </div>
+    <div sx={{
+      display: "flex",
+      flexDirection: "row",
+      '@media screen and (max-width: 700px)': {
```

Comment:
Adding in all of the responsiveness code here for the profile header so it plays nicely on mobile.